Catappult/appcoins-wallet-android
# Feature/APPC-2706 Hold NFTs

**What does this PR do?**

   Implements NFT in wallet

**How should this be manually tested?**

  To test we need a wallet with NFTs, then go to my wallets and click in the NFTs button.
  Is expected to list the NFTs associated with the wallet and when we click on the NFT to be able to see the details. When the wallet has no NFTs is expected to present the No NFTs screen and if the user has no Internet connection it will show the reconnect screen.
Please test the following use cases: 
When the wallet has a NFT with no image
When the wallet has a NFT and the image is a GIF

**What are the relevant tickets?**

  Tickets related to this pull-request: [APPC-2706](https://aptoide.atlassian.net/browse/APPC-2706)

**Code Review Checklist**
